src/main/scala/com/databricks/spark/redshift/RedshiftRelation.scala

```diff
@@ -179,7 +179,12 @@ private[redshift] case class RedshiftRelation(
     // the credentials passed via `credsString`.
     val fixedUrl = Utils.fixS3Url(Utils.removeCredentialsFromURI(new URI(tempDir)).toString)
 
-    s"UNLOAD ('$query') TO '$fixedUrl' WITH CREDENTIALS '$credsString' ESCAPE MANIFEST"
+    val userUnloadOptions = params.parameters.getOrElse("extraunloadoptions", "").toUpperCase()
```

Comment:
I'm not sure that you necessarily want to automatically convert this to upper-case; do you remember why this was necessary?